packages/graphql-server/src/functions/graphql.ts

```diff
@@ -41,8 +35,8 @@ import type { GraphQLHandlerOptions } from './types'
  *
  * Unexpected errors are those that are not Envelop, GraphQL, or Redwood errors
  **/
-export const formatError: FormatErrorHandler = (err: any, message: string) => {
-  const allowErrors = [GraphQLYogaError, EnvelopError, RedwoodError]
+export const formatError: MaskError = (err: any, message: string) => {
+  const allowErrors = [RedwoodGraphQLError, RedwoodError]
```

Comment:
Note to myself, we should add a test to confirm the behavior of error masking.

Reply:
We dropped EnvelopError and GraphQLYogaError in v3 of Yoga and Envelop so maybe we can have more tests on your end.